pages/feedback_review.py

```
@@ -0,0 +1,233 @@
+    @staticmethod
+    def _render_metrics() -> None:
+        """Live learning metrics dashboard."""
+        st.header("📊 Learning Metrics & Trends")
+
+        try:
+            from security_engine.adaptive_memory import FeedbackPinner
+        except ImportError:
+            st.error("❌ Memory module not available")
+            return
+
+        memory = FeedbackPinner()
```

Ruff (0.14.10)

Local variable `memory` is assigned to but never used

Remove assignment to unused variable `memory`

(F841)